mautic/mautic
# Company name malformed

| Q | A |
| --- | --- |
| Bug fix? | Y |
| New feature? |  |
| Related user documentation PR URL |  |
| Related developer documentation PR URL |  |
| Issues addressed (#s or URLs) | #2866 |
| BC breaks? |  |
| Deprecations? |  |
#### Description:

When importing a contact, company name are HTML encoded into the mysql database
#### Steps to test this PR:

see #2866. At the last step, you should see the correct company name in your mysql database
#### Steps to reproduce the bug:

see #2866
